src/molecules/modal/Modal.selectors.json

```diff
@@ -2,5 +2,21 @@
   {
     "name": ".a-modal--blank",
     "description": "Remove default content paddings"
+  },
+  {
+    "name": ".a-modal--compact",
```

Comment:
do we need this modifier?

Reply:
It was taken from the old code, I can't find in designs what was the reason of this modifier so I'll remove it - if needed in future then we can easily add appropriate styles on higher component level